Integrations/PaloAltoNetworks_Traps/PaloAltoNetworks_Traps.py

```diff
@@ -0,0 +1,550 @@
+
+
+def test_module_command():
+    health_check()
+    res = http_request('GET', 'agents/1', parse_response=False)
+    if res.status_code == 403:
+        return_error(f'Error connecting to server. Check your Application ID and Private key')
+    return
+
+
+def get_endpoint_by_id_command():
+    args = demisto.args()
+    endpoint_id = args.get('endpoint_id')
+    endpoint_data, raw_data = get_endpoint_by_id(endpoint_id)
+    md = tableToMarkdown(f'Endpoint {endpoint_id} data:', endpoint_data, headerTransform=pascalToSpace)
+    context = {'Traps.Endpoint(val.ID == obj.ID)': createContext(endpoint_data)}
+    return_outputs(md, context, raw_response=raw_data)
+
+
+def endpoint_files_retrieve_command():
+    args = demisto.args()
+    endpoint_id = args.get('endpoint_id')
+    file_name = args.get('file_name')
+    event_id = args.get('event_id')
+    operation_obj = endpoint_files_retrieve(endpoint_id, file_name, event_id)
+    md = tableToMarkdown(f'Files retrieve command on endpoint: {endpoint_id} received', operation_obj,
+                         headerTransform=pascalToSpace)
+    context = {'Traps.FileRetrieve(val.OperationID == obj.OperationID)': operation_obj}
+    return_outputs(md, context)
+
+
+def endpoint_files_retrieve_result_command():
+    args = demisto.args()
+    operation_id = args.get('operation_id')
+    status_obj = endpoint_files_retrieve_result(operation_id)
+    md = f'### File retrieval status is: {status_obj.get("Status")}'
+    context = {'Traps.FileRetrieveResult(val.OperationID == obj.OperationID)': status_obj}
+    return_outputs(md, context)
+
+
+def endpoint_scan_command():
+    args = demisto.args()
+    endpoint_id = args.get('endpoint_id')
+    operation_obj = endpoint_scan(endpoint_id)
+    md = tableToMarkdown(f'Scan command on endpoint: {endpoint_id} received', operation_obj,
+                         headerTransform=pascalToSpace)
+    context = {'Traps.Scan(val.OperationID == obj.OperationID)': operation_obj}
+    return_outputs(md, context)
+
+
+def endpoint_scan_result_command():
+    args = demisto.args()
+    operation_id = args.get('operation_id')
+    status_obj = endpoint_scan_result(operation_id)
+    context = {f'Traps.ScanResult(val.OperationID == obj.OperationID)': status_obj}
+    md = tableToMarkdown(f'Status of scan operation: {operation_id}', status_obj, headerTransform=pascalToSpace)
+    return_outputs(md, context)
+
+
+def event_update_command():
+    args = demisto.args()
+    event_id = args.get('event_id')
+    status = args.get('status')
+    comment = args.get('comment')
+    event_update_status_and_command(event_id, status, comment)
+    md = f'### Event: {event_id} was updated'
+    md += f'\n##### New status: {status}' if status else ''
+    md += f'\n##### New comment: {comment}' if comment else ''
+    return_outputs(md, {})
+
+
+def event_bulk_update_status_command():
+    args = demisto.args()
+    event_ids = args.get('event_ids').split(',')
```

Comment:
event_ids could be a list also, that is why we need to use argToList
```suggestion
    event_ids = argToList(args.get('event_ids'))
```

Reply:
- [x] Fixed